recipes/OpenVisus/meta.yaml

```
@@ -0,0 +1,59 @@
+package:
+  name: openvisus
+  version: 1.3.3
+
+# to debug you can use inplace code
+# do a 'rm -Rf ~/Desktop/OpenVisus/build && rm -Rf ~/anaconda3/conda-bld/openvisus*'
+# source:
+#  path: ../..
+
+source:
+  git_rev: 1.3.3
+  git_url: https://github.com/sci-visus/OpenVisus.git
```

http://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos

```suggestion
  url: https://github.com/sci-visus/OpenVisus/archive/{{ version }}.tar.gz
  sha256: d9e706bf09a1b809dc99e2f7101ed8bd4e7b5b042927c020cf0785c23e41b6f1
```

Also remove the `git_rev` entry above.